### PR TITLE
auto OCP-33743

### PR DESCRIPTION
**Pull request by yapei**
Ready for review @pruan-rht  @liangxia  @yanpzhan  Please help review

### CODE REVIEW

File: features/step_definitions/catalog_source.rb [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+    step %Q/admin ensures "#{cs_name}" catalog_source is deleted from the "openshift-marketplace" project/
+  }
+
+  step %Q/I process and create:/, table(%{
+    | f | #{BushSlicer::HOME}/testdata/olm/catalogsource-template.yaml |
+    | p | NAME=#{cs_name}                                              |
+    | p | IMAGE=#{cs_image}                                            |
+  })
+  raise "Error creating catalogsource" unless @result[:success]
+  step %Q/I use the "openshift-marketplace" project/
```
Comment (liangxia) [inline features/step_definitions/catalog_source.rb:16]: Duplicate with line 4

File: features/step_definitions/catalog_source.rb [hunk 2]
```diff
@@ -0,0 +1,21 @@
+Given(/^admin creates "([^"]*)" catalog source with image "([^"]*)"$/) do |cs_name, cs_image|
+  ensure_admin_tagged
+  step %Q/I switch to cluster admin pseudo user/
+  step %Q/I use the "openshift-marketplace" project/
+
+  teardown_add {
+    step %Q/admin ensures "#{cs_name}" catalog_source is deleted from the "openshift-marketplace" project/
+  }
```
Comment (liangxia) [inline features/step_definitions/catalog_source.rb:8]: Perhaps `step %Q/admin ensures "#{cs_name}" catalog_source is deleted from the "openshift-marketplace" project after scenario/`.
Current one also works.

File: features/step_definitions/catalog_source.rb (hunk 1 above)
Comment (yapei) [reply features/step_definitions/catalog_source.rb:16]: yeah, let me remove this line

File: features/step_definitions/catalog_source.rb (hunk 2 above)
Comment (yapei) [reply features/step_definitions/catalog_source.rb:8]: yeah, maybe both is ok

Comment (yapei) [reply features/step_definitions/catalog_source.rb]: updated based on your comments

File: features/step_definitions/catalog_source.rb (hunk 1 above)
Comment (yapei) [reply features/step_definitions/catalog_source.rb]: updated

**Comment by yapei** [comment]
requested changes updated @pruan-rht  @liangxia  Can you help PTAL again?

**Pull request merged**